snap/README.md

```diff
@@ -0,0 +1,13 @@
+# Build
+
+Install `snapcraft`: `sudo apt install snapcraft`
+
+Changedir into `snap`: `cd snap`
+
+Build `snap/strictdoc_0.0.18_amd64.snap`: `snapcraft`
+
+## Installation
```

Comment:
As soon as PR is merged the installation can be simplified to:

```
wget https://github.com/strictdoc-project/strictdoc/raw/snap/snap/strictdoc_0.0.18_amd64.snap
sudo snap install strictdoc_*.snap --devmode --dangerous
```

Reply:
Could you also provide the documentation in the `docs/strictdoc.sdoc`? Please also address this comment at the same time: https://github.com/strictdoc-project/strictdoc/pull/572#issuecomment-1003721895. This way, you will add two new installation methods to the documentation.

Reply:
I've added the snap installation part in this PR. I'll address #572 in a follow up PR to keep topics separated. Do you want squashed PRs or is it ok to leave the single commits of this PR as they are?

Reply:
Please address the #572 separately. Generally, I would prefer 1 commit by 1 PR with everything rebased every time. But this one I will merge as is :+1: